src/views/onboarding/Onboarding.tsx

```diff
@@ -187,6 +305,14 @@ function OnboardingComponent({ productId }: { productId: string }) {
         request_id: requestIdRef.current,
         product_id: productId,
       });
+      if ((postLegalsResponse as AxiosError<Problem>).response?.status === 403) {
+        trackEvent('ONBOARDING_NOT_ALLOWED_ERROR', {
+          party_id: externalInstitutionId,
+          request_id: requestIdRef.current,
+          product_id: productId,
+        });
+        setOutcome(notAllowedError);
```

Comment:
in this way, you are not invoking the setOutcome to set the "error" OutcomeMessage